test/membrane/async-await.spec.js

```diff
@@ -0,0 +1,16 @@
+import createSecureEnvironment from "../../lib/browser-realm.js";
+
+describe("async/await", () => {
+    it("basic wrapping", (done) => {
+        const evalScript = createSecureEnvironment({ endowments: { done, expect }});
+        evalScript(`
+            async function hello() {
+                return await "Hello";
+            }
+            hello().then((value) => {
+                expect(value).toBe("Hello");
+                done();
+            });
+        `);
+    });
```

Comment:
Here the test will never complete because the fail with async/await in Firefox/Safari is that it never resolves so doesn't call `then()`.

Reply:
Yes, it times out.

Reply:
```
Firefox 80.0.0 (Mac OS X 10.15.0) async/await basic wrapping FAILED
        Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL) in node_modules/jasmine-core/lib/jasmine-core/jasmine.js (line 6350)
        <Jasmine>
Firefox 80.0.0 (Mac OS X 10.15.0): Executed 82 of 82 (1 FAILED) (7.132 secs / 7.039 secs)
```